src/main/java/org/jitsi/videobridge/health/Health.java

```diff
@@ -272,23 +234,9 @@ private static String generateEndpointID()
      * Iniatializes a new {@link Health} instance for a specific
      * {@link Videobridge}.
      */
-    public Health(Videobridge videobridge, ConfigurationService cfg)
+    public Health(Videobridge videobridge)
     {
-        super(videobridge, HealthIntervalProperty.getValue(), true);
-
-        if (cfg == null)
-        {
-            logger.warn("Configuration service is null, using only defaults.");
-        }
-
-        timeout =
-            cfg == null ? TIMEOUT_DEFAULT
-                : cfg.getInt(TIMEOUT_PNAME, TIMEOUT_DEFAULT);
-
-        stickyFailures
-            = cfg == null ? STICKY_FAILURES_DEFAULT
-                : cfg.getBoolean(
-                    STICKY_FAILURES_PNAME, STICKY_FAILURES_DEFAULT);
+        super(videobridge, HealthConfig.HealthIntervalProperty.getValue(), true);
```

Comment:
Shouldn't this use `HealthConfig.getInterval()`?

Reply:
yes, it should, nice catch.  maybe it'd be best to make these classes protected (so just the tests can access it to test directly)